lms/djangoapps/grades/admin.py

```diff
@@ -26,6 +27,49 @@ class CoursePersistentGradesAdmin(KeyedConfigurationModelAdmin):
         }),
     )
 
+
+class PersistentSubsectionGradeOverrideAdmin(admin.ModelAdmin):
+    fieldsets = (
+        (None, {
+            'fields': (
+                'grade',
+                'earned_all_override',
+                'earned_graded_override',
+                'possible_all_override',
+                'possible_graded_override',
+            ),
+            'description': 'Enter the ID of the subsection grade you want to override. You will probably need to '
+                           'find this in the read replica in the grades_persistentsubsectiongrade table.'
+        }),
+    )
+    list_display = [
+        'get_course_id',
+        'get_usage_key',
+        'get_user_id',
+        'earned_all_override',
+        'earned_graded_override',
+        'created',
+        'modified',
+    ]
+    list_filter = ('grade__course_id', 'grade__user_id', 'grade__usage_key')
```

Comment:
<img width="833" alt="screen shot 2017-11-23 at 12 23 46 am" src="https://user-images.githubusercontent.com/4252738/33146059-9e001022-cfe4-11e7-94da-d0c3fd447f6e.png">

@sanfordstudent filter by `grade__usage_key` can lists a large list when the model starts filling up + it causes the display out of order. Do you think sticking with the filter key is right when we can aleady search by `grade__usage_key`

Reply:
Oh good catch. I will remove that filter. Thank you!